dashboard/modules/job/common.py

```diff
@@ -85,10 +91,20 @@ class JobInfo:
     def __post_init__(self):
         if self.message is None:
             if self.status == JobStatus.PENDING:
-                self.message = (
-                    "Job has not started yet, likely waiting "
-                    "for the runtime_env to be set up."
-                )
+                self.message = "Job has not started yet."
+                if any(
+                    [
+                        self.num_cpus is not None and self.num_cpus > 0,
+                        self.num_gpus is not None and self.num_gpus > 0,
+                        self.resources not in [None, {}],
+                    ]
+                ):
+                    self.message += " It may be waiting for resources "
+                    "(CPUs, GPUs) to become available."
```

Comment:
it'd be very nice to add the required resources and currently available resources in the cluster to this message (similar to the one that we log in the serve controller)

Reply:
Agreed. I didn't see an quick way to add this, but made a followup issue here https://github.com/ray-project/ray/issues/29921